src/shared/components/DismissibleMessage/DismissibleMessage.style.ts

```diff
@@ -0,0 +1,46 @@
+import { colors, sizes } from '@/shared/theme'
+import styled from '@emotion/styled'
+import Text from '../Text'
+
+export const MessageWrapper = styled.div`
+  position: relative;
+  margin: ${sizes(6)} 0;
```

Comment:
```suggestion
```

We probably shouldn't have a margin on the message component itself, as this will vary based on where its used. The view can supply that itself